gbpservice/tests/contrib/diskimage-create/disk_image_create.py

```diff
@@ -31,6 +31,11 @@ def parse_json(j_file):
         conf = jsonutils.load(json_data)
     return
 
+def set_nfp_git_branch():
+    Dockerfile_path="/opt/stack/gbp/gbpservice/nfp/configurator/Dockerfile"
```

Comment:
As discussed, correct this. Pass local.conf as argument to disk image create script and get the GIT branch details.